company_website/static/common/scripts/head/google_tag_manager.js

```diff
@@ -0,0 +1,10 @@
+(function(w, d, s, l, i){
```

Comment:
Please at least use variables shortcuts instead of one letter naming

Reply:
Crikey, I don't even no what half of them are! They just give you that nasty HTML-nested minimised script and I can't find any documentation or any kind of breakdown for it anywhere! 😩 
It's all this "Copy and paste rady, bro! No need to know what's all this for, trust me!" BS

Reply:
They are probably from left to right: window, document, script, layer, id :)

Reply:
Yeah, super silly of me not to check the function call at the bottom. 😖